CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ cmake_minimum_required (VERSION 3.10)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake-modules")
 
+include(CMakePackageConfigHelpers)
```

Comment:
Why do we need this?

Reply:
Not sure why it is added  XD...
Need to revert this.. sorry

Reply:
my master branch on my fork got corrupt with unwanted changes. Sorry about it